spec/requests/portfolios_spec.rb

```diff
@@ -52,6 +52,24 @@
       end
     end
 
+    describe "POST admin tagged /portfolios/:portfolio_id/portfolio_items" do
```

Comment:
@lindgrenj6 - Were these specs added due to coverage issues? I like the specs but not sure they belong in this PR

Reply:
Moving this one to its own PR, it was kind of an extra one I noticed we didn't have.